master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -29,10 +29,18 @@ We'll use a new namespace for this guide.  Run the following command to create i
 kubectl create ns advanced-policy-demo
 ```
 
-And then enable isolation on the Namespace.
+And then enable isolation on the Namespace using a [default policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-policies).
 
 ```
-kubectl annotate ns advanced-policy-demo "net.beta.kubernetes.io/network-policy={\"ingress\":{\"isolation\":\"DefaultDeny\"}}"
+kubectl create -f - <<EOF
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
```

Comment:
Yeah, I think we should leave as is and then raise an issue to address once v1.8 is released - that should give enough time such that this doesn't throw off a whole bunch of users.